modules/navigation.js

```diff
@@ -0,0 +1,39 @@
+const navigation = () => {
+  const bookListLink = document.getElementById('book-lists');
+  const bookFormLink = document.getElementById('book-form');
+  const contactInfoLink = document.getElementById('contact-info');
+  const bookSection = document.querySelector('.books-section');
+  const formSection = document.querySelector('.form-section');
+  const contactSection = document.querySelector('.contact-section');
+
+  bookListLink.addEventListener('click', () => {
+    bookListLink.classList.add('active');
+    bookFormLink.classList.remove('active');
+    contactInfoLink.classList.remove('active');
+
+    bookSection.classList.remove('invisible');
+    formSection.classList.add('invisible');
+    contactSection.classList.add('invisible');
+  });
+
+  bookFormLink.addEventListener('click', () => {
+    bookListLink.classList.remove('active');
+    bookFormLink.classList.add('active');
+    contactInfoLink.classList.remove('active');
+    bookSection.classList.add('invisible');
+    formSection.classList.remove('invisible');
+    contactSection.classList.add('invisible');
+  });
+
+  contactInfoLink.addEventListener('click', () => {
+    bookListLink.classList.remove('active');
+    bookFormLink.classList.remove('active');
+    contactInfoLink.classList.add('active');
+    contactInfoLink.classList.add('active');
+    bookSection.classList.add('invisible');
+    formSection.classList.add('invisible');
+    contactSection.classList.remove('invisible');
+  });
+};
```

Comment:
- [ ] I am unable to visit different pages in your app. Regardless of which navbar menu I select, this is the screen it keeps showing me (see image). Kindly fix that. I am unable to see the form for adding a book, so I cannot check if books are listed or if the contact is displayed.

  ![image](https://user-images.githubusercontent.com/36542986/212901496-b0f8c915-28b9-4c76-bfd6-38e2a8dea35b.png)

Reply:
@TedLivist 

https://user-images.githubusercontent.com/56417904/212906367-55834743-3e0c-415c-8708-ade5932c8a76.mov

Everything is working smoothly on my end. Could you double check that you're viewing the correct branch ('awesome-books-ES6') and using live server? My partner has also confirmed that it's functioning properly on their end as well, as shown in the video. Let me know if you're still experiencing any issues. Thank you.